src/Uno.UI/UI/Xaml/Media/VisualTreeHelper.cs

```diff
@@ -487,19 +487,23 @@ internal static IReadOnlyList<_View> ClearChildren(UIElement view)
 				TRACE($"- renderTransform: {tr.ToMatrix(element.RenderTransformOrigin, element.ActualSize.ToSize())}");
 
 #if __SKIA__
-			var transformToElement = element.TransformToVisual(null);
+			var transformToElement = UIElement.GetTransform(element, null);
```

Comment:
`TransformToVisual` will allocate `MatrixTransform` unnecessarily, which is simply just a wrapper around `Matrix` (being a DP).

Using `GetTransform` to return `Matrix` right away is more performant in two ways:

1. Avoids the `MatrixTransform` allocation
2. Avoids calling into the DP system when setting `MatrixTransform.Matrix` dependency property, which then calls into a property changed callback